frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/SoftwareInstallerCard/InstallerPoliciesTable/InstallerPoliciesTableConfig.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import { ISoftwareInstallPolicy } from "interfaces/software";
+import PATHS from "router/paths";
+import { getPathWithQueryParams } from "utilities/url";
+
+import LinkCell from "components/TableContainer/DataTable/LinkCell";
+import HeaderCell from "components/TableContainer/DataTable/HeaderCell";
+
+interface IInstallerPoliciesTableConfig {
+  teamId?: number;
+}
+interface ICellProps {
+  cell: {
+    value: string;
+  };
+  row: {
+    original: ISoftwareInstallPolicy;
+  };
+}
+
+const generateInstallerPoliciesTableConfig = ({
+  teamId,
+}: IInstallerPoliciesTableConfig) => {
+  const tableHeaders = [
+    {
+      accessor: "name",
+      disableSortBy: false, // TODO: Sorting works, but arrows don't show sorting
```

Comment:
Noting the TODO.

tbh sorting by name doesn't do much for us for a short policy list, so IMO we should drop sorting entirely here to decrease visual noise. @eugkuo WDYT?

Reply:
Fixed it, it works, @eugkuo lmk if I should remove it

Reply:
FWIW given that I have capacity this sprint, seems like the solution here is "fix #28687 and keep this"